.github/workflows/lint.yml

```diff
@@ -39,7 +39,7 @@ jobs:
       - uses: actions/checkout@v4
       - uses: dtolnay/rust-toolchain@nightly
         with:
-          toolchain: nightly-2024-09-09
+          toolchain: nightly
```

Comment:
can just remove, the action defaults to nightly (`@nightly`)

Reply:
ture, thanks